test-files/algorithms/wayjoiner/WayJoinerAdvancedConflateExpected.osm

```diff
@@ -150,23 +150,23 @@
         <nd ref="-34"/>
         <nd ref="-33"/>
         <nd ref="-30"/>
+        <tag k="hoot:status" v="3"/>
         <tag k="highway" v="road"/>
         <tag k="note" v="2"/>
-        <tag k="hoot:status" v="3"/>
-        <tag k="error:circular" v="15"/>
         <tag k="hoot:id" v="-192"/>
+        <tag k="error:circular" v="15"/>
```

Comment:
So much of this from here down is reordering tags that probably should have been reordered back when the Qt5 changes actually reordered them.  `hoot diff` doesn't care about the order but when more ways are joined back together it gets really cluttered.